feature-libs/product-configurator/rulebased/components/overview-form/configurator-overview-form.component.html

```diff
@@ -3,7 +3,14 @@
     <ng-container *ngFor="let group of configuration.overview.groups">
       <div class="cx-group">
         <h2>
-          {{ group.groupDescription }}
+          <!-- TODO: update feature level to equals 3.1 without exclamation mark '!' -->
+          <ng-container *cxFeatureLevel="!3.1">
+            {{ group.groupDescription }}
+          </ng-container>
+          <!-- TODO: update feature level to current release version -->
+          <ng-container *cxFeatureLevel="3.1">
```

Comment:
In addition: Also your solution does not work for me. For level '3.1' or '3.4' in app.module.ts, always the second version (with the `<span>`) is taken. I don't get a different behaviour.

Reply:
Ok, got it. With 3.0 it is working fine. 
But the comment instruction is wrong from my point of view, submitted another one.